packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.html

```diff
@@ -0,0 +1,102 @@
+<div>
+    <oui-back-button data-on-click="$ctrl.goBack()"
+        ><span data-translate="dedicatedCloud_manage_nsx_edge_title"></span>
+    </oui-back-button>
+</div>
+
+<p
+    class="my-4"
+    data-translate="dedicatedCloud_tab_datacenter_global_information"
+></p>
```

Comment:
Maybe you can use oui-header from ui-kit
```suggestion
<oui-header
      data-description="{{ 'dedicatedCloud_tab_datacenter_global_information' | translate }}">
      <oui-back-button
        data-on-click="$ctrl.goBack()"
        data-previous-page="{{ 'dedicatedCloud_manage_nsx_edge_title' | translate }}"
        >
      </oui-back-button>
    </oui-header>
```

Reply:
Oh nice thanks ! I didn't think about it at first